tests/utils/test_cli_util.py

```diff
@@ -185,6 +185,47 @@ def test_get_dag_by_pickle(self, session, dag_maker):
         with pytest.raises(AirflowException, match="pickle_id could not be found .* -42"):
             get_dag_by_pickle(pickle_id=-42, session=session)
 
+    @pytest.mark.parametrize(
+        ["given_command", "expected_masked_command"],
+        [
+            (
+                "airflow variables set --description 'needed for dag 4' client_secret_234 7fh4375f5gy353wdf",
+                "airflow variables set --description 'needed for dag 4' client_secret_234 ********",
+            ),
+            (
+                "airflow variables set cust_secret_234 7fh4375f5gy353wdf",
+                "airflow variables set cust_secret_234 ********",
```

Comment:
Let us add the description field too, otherwise it gives an intuition that masking removes description.

Reply:
This is different test case, I am pointing too. I have included both type of cli commands which users can use to set variables, and the masking should be done on both types.

Reply:
My bad.